### PR TITLE
English: search

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Activate search in english

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -132,6 +134,14 @@ en:
     title: "Products that we are proud of"
     capp:
       capp-lms: "CAPP Learning & Developing"
+  search:
+    heading: Search
+    placeholder: Search query
```
Comment (sn3p) [inline locales/en.yml]: Maybe `Search our website` sounds a bit more friendly?

File: locales/en.yml [hunk 2]
```diff
@@ -132,6 +134,14 @@ en:
...
+    result: "result"
+    results: "results"
+    downloading: "Download index"
```
Comment (sn3p) [inline locales/en.yml]: `Downloading index...` would be better as the status.

File: source/localizable/zoeken.html.erb [hunk 3]
```diff
@@ -2,9 +2,11 @@
 title:
   nl: Zoeken
   de: Suche
+  en: Search
 description:
   nl: Zoek in onze website
   de: Durchsuchen Sie unsere Website
+  en: Search on our website
```
Comment (sn3p) [inline source/localizable/zoeken.html.erb]: `Search our website`

**Comment by RonaldDijkstra** [comment]
Thanks

**Pull request merged**